joomla/joomla-cms
# [4.1] com_scheduler checked out tasks

Pull Request for Issue #36676

### Summary of Changes

- Show information about "checked out" state in tasks list
- Add button to checkin tasks

### Testing Instructions

1. Open an existing task
2. Go back to the list view (without "Close")
3. Check informations about "checked-out" state
4. "Check in" the task by using the button in the toolbar or the lock icon
5. Check the _Maintenance: Global Check-in_

### Actual result BEFORE applying this Pull Request
#### Open task edit
![image](https://user-images.githubusercontent.com/66922325/151413958-702759a9-da89-4d47-b165-ba9ef693558e.png)
![image](https://user-images.githubusercontent.com/66922325/151414202-7f15fe92-e8a7-494f-ace2-586e41e1e4f0.png)
![image](https://user-images.githubusercontent.com/66922325/151414364-1d1451fc-3c39-4eaf-b7a4-c7ce90bfc495.png)
#### Close task edit
![image](https://user-images.githubusercontent.com/66922325/151415507-5a1fdc24-a281-4fee-9c31-f19f79fbae47.png)
![image](https://user-images.githubusercontent.com/66922325/151415794-04041212-9bc5-41ef-8ddb-2cc8c132c4fd.png)

### Expected result AFTER applying this Pull Request
#### Open task edit
![image](https://user-images.githubusercontent.com/66922325/151413958-702759a9-da89-4d47-b165-ba9ef693558e.png)
![image](https://user-images.githubusercontent.com/66922325/151414907-382858c0-5886-4573-a100-796b81eac26e.png)
![image](https://user-images.githubusercontent.com/66922325/151414990-f3bfb051-c366-4cb8-921b-cf7e13a7f478.png)
#### Close task edit
![image](https://user-images.githubusercontent.com/66922325/151415987-1a9c2db3-9f61-4904-8bb0-0b4cc8801fd5.png)
![image](https://user-images.githubusercontent.com/66922325/151416130-e001ee0e-d867-48ef-afc8-2e067c16b33b.png)

### Documentation Changes Required

No documentation yet
